gnocchi/tests/functional/test_gabbi.py

```diff
@@ -28,9 +32,8 @@
 PREFIX = '/gnocchi'
 
 
-def load_tests(loader, tests, pattern):
-    """Provide a TestSuite to the discovery process."""
+def pytest_generate_tests(metafunc):
     test_dir = os.path.join(os.path.dirname(__file__), TESTS_DIR)
-    return driver.build_tests(test_dir, loader, host=None, prefix=PREFIX,
-                              intercept=fixtures.setup_app,
-                              fixture_module=fixtures)
+    driver.py_test_generator(test_dir, prefix=PREFIX,
+                             intercept=fixtures.setup_app,
```

Comment:
@cdent any idea http://paste.openstack.org/show/622670/ ?

Looks like gabbi does not call fixtures.setup_app()

Reply:
Zounds, moving to pytest, that's exciting. It's supposed to do fixtures, so I'm not sure what's going on, I'll pull this down and try it myself.